kufar/books/main.go

```diff
@@ -70,12 +76,23 @@ type CredentialsRequest struct {
 	Password string `json:"password"`
 }
 
+//Cache ...
+type Cache struct {
+	// defaultExpiration time.Duration
+
+	mu    sync.RWMutex
+	Items map[int]Book
+}
+
+type Cacher interface {
+	get(int) (Book, bool)
```

Comment:
It's a bit strange to create an interface with private methods :)
Let's use public methods instead.